Roles/Impostor/DollMaster.cs

```diff
@@ -352,51 +355,33 @@ private static bool CanKillerUseAbility(PlayerControl player)
     }
 
     // A fix when the DollMaster or Possessed Player DC's from the game.
-    // Untested!
-    /*
-    public override void OnPlayerLeft(InnerNet.ClientData clientData)
+    // Runing in PlayerContolPatch.cs
+    public static void CheckIfPlayerDC()
     {

```

Comment:
Better to use `CustomRoleManager.OnFixedUpdateLowLoadOthers.Add(CheckIfPlayerDC);`

Like here:
![image](https://github.com/0xDrMoe/TownofHost-Enhanced/assets/104814436/aec533b9-d04b-41be-95b1-9c6872f086bd)
![image](https://github.com/0xDrMoe/TownofHost-Enhanced/assets/104814436/125aa4e3-b21d-4527-a6f2-46ac440f794c)

Reply:
And then remove it: 
![image](https://github.com/0xDrMoe/TownofHost-Enhanced/assets/104814436/41f929ff-de8f-4ef1-b416-93966183246f)